tests/test_data_importer.py

```diff
@@ -11,33 +11,6 @@
 TRG = "en"
 CURRENT_FOLDER = os.path.dirname(os.path.abspath(__file__))
 
-hplt_translations = {
```

Comment:
This is fragile and hard to update. Also checking the first 10 lines is limited, so I replaced it with checking max line length for the whole dataset.

Reply:
I would prefer to keep it, as you can copy and paste the new values in there, otherwise you don't know if it's doing the correct thing. This shows the behavior of the code, where otherwise mistakes can happen. For instance in code review I don't know what the new behavior is now, since this would demonstrate a behavior change.

Reply:
ok, I can return it, but I need to change the structure so that it's easier to copy-paste it from a file